internal/controller/monitor/monitor.go

```diff
@@ -103,40 +99,58 @@ func (m *Monitor) Start(ctx context.Context, handler JobHandler) <-chan error {
 	logger := m.logger.With(zap.String("org", m.cfg.Org))
 	errs := make(chan error, 1)
 
+	var agentTags map[string]string
+	{
+		var tagErrs []error
+		if agentTags, tagErrs = agenttags.ToMap(m.cfg.Tags); len(tagErrs) != 0 {
+			logger.Warn("making a map of agent tags", zap.Errors("err", tagErrs))
+		}
+	}
```

Comment:
What about:

```suggestion
	agentTags, tagErrs := agenttags.ToMap(m.cfg.Tags)
	if len(tagErrs) != 0 {
		logger.Warn("making a map of agent tags", zap.Errors("err", tagErrs))
	}
```

Reply:
I don't want the `tagErrs` value to leak outside this scope. I personally don't think using curly braces to isolate variables' scopes is unusual, but I suppose it's pretty cheap to put it into a function to avoid raising any eyebrows, though, so I will do that instead.

Reply:
It's gone now